### ISSUE TITLE
Can connect a Bluetooth keyboard but can't connect a Bluetooth mouse

### ISSUE THREAD

**Issue by Duxter7** [daliansky/XiaoMi-Pro-Hackintosh#233]
So I i have the latest clover and macOS Mojave 10.14.6. Installed it using the latest Installation guide. The problem is i have working bluetooth, it shows devices and i can connect to some of them (Logitech K380) but i can't connect neither Microsoft Designer Mouse, Mi8 phone nor my desktop PC. I also tried disabling internal bluetooth and plugging external Asus BT dongle which i use on my desktop pc. Now i can turn BT on and off but it's the same thing. I can't connect my mouse. 

Basically it tries to connect, then says connected and then disconnects.

When using ASUS BT it says only "connecting" and then nothing/

Am i doing something wrong?

**Comment by wisper-shadow** [comment]
It seems that MacOS only support Apple Magic Mouse, because I also tried to connect my Mi Bluetooth Mouse and encounter the same problem of yours. But when I switch to Apple Magic Mouse, it seems that it works well.

**Comment by Duxter7** [comment]
> @Duxter7 It seems that MacOS only support Apple Magic Mouse, because I also tried to connect my Mi Bluetooth Mouse and encounter the same problem of yours. But when I switch to Apple Magic Mouse, it seems that it works well.

No, it’s not that. My GF uses the same Microsoft Designer mouse on her MacBook Pro 13 2017 and it works perfectly

**Comment by influenist** [comment]
@wisper-shadow all my Mi Bluetooth Mouses (I have 4) are hardly connecting with any Hackintosh (let me try my geniune MacBook later today). I don't think MacOS is rejecting BT mouses in general, rather a BT compatibility issue. Tomorrow I receive a Satechi M1 mouse to match up my BT keyboard of them. I'll keep you posted if it works.

**Comment by stevezhengshiqi** [comment]
It's common that Intel Bluetooth doesn't support Mi Bluetooth Mouses, use Broadcom bluetooth for better experience. The issue is more likely related to Linux Intel BT firmware and it's hard to solve.

**Comment by stevezhengshiqi** [comment]
It’s an IntelBluetoothFirmware side issue.

**Comment by Erix223** [comment]
My Bluetooth keyboard (Microsoft Designer Keyboard) is experiencing the same issue with Bluetooth pairing. I suspect it may be related to an Intel Wi-Fi card (AX210).
